lmfit/model.py

```diff
@@ -1409,6 +1411,11 @@ def fit(self, data=None, params=None, weights=None, method=None,
                 except AttributeError:
                     pass
 
+        if self.data is not None and len(self.data) > 1:
+            sstot = max(tiny, ((self.data - self.data.mean())**2).sum())
+            if isinstance(self.residual, np.ndarray) and len(self.residual) > 1:
+                self.rsquared = 1.0 - (self.residual**2).sum() / sstot
```

Comment:
for readability I would prefer the `max(tiny, .....)` to be in the line above instead of the `sstot` define, but that might be a personal preference

Reply:
@reneeotten Yeah, I agree.  I think I was fighting line length considerations, but readability is not only line length ;)